packages/matchbox/src/components/Pager/Next.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ArrowForward } from '@sparkpost/matchbox-icons';
+import { Button } from '../Button';
+import styles from './Pager.module.scss';
+
+const Next = ({ disabled, onClick }) => (
```

Comment:
Instead of specifying `onClick` maybe just pass through `...rest`, so we get support for `component, size, etc`

Actually can probably just pass through all props, without specifying disabled

Reply:
unless you think we don't need anything else

Reply:
I don't think `component` will be needed, but definitely size, color, etc.